<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PBL</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Разпознование текста</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="save_btn">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Сохранить</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>450</width>
                <height>350</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Verdana</family>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"> QLabel{border: 2px solid #1E90FF}   </string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Добавьте изображение&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="scanbutton">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Сканировать (rus+eng)</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="copy_text_btn">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Скопировать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Сохранения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="LoudAction">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="load"/>
    <addaction name="paste"/>
   </widget>
   <widget class="QMenu" name="lang">
    <property name="title">
     <string>Язык сканирования</string>
    </property>
    <addaction name="rus_eng"/>
    <addaction name="rus"/>
    <addaction name="eng"/>
   </widget>
   <widget class="QMenu" name="translate">
    <property name="title">
     <string>Перевести</string>
    </property>
    <addaction name="tr_eng"/>
    <addaction name="tr_rus"/>
   </widget>
   <widget class="QMenu" name="menu_resize">
    <property name="title">
     <string>Изменит картинку</string>
    </property>
    <addaction name="act_resize"/>
    <addaction name="right_90"/>
    <addaction name="left_90"/>
   </widget>
   <addaction name="LoudAction"/>
   <addaction name="menu_resize"/>
   <addaction name="lang"/>
   <addaction name="translate"/>
  </widget>
  <action name="load">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="right">
   <property name="text">
    <string>Вправо</string>
   </property>
  </action>
  <action name="left">
   <property name="text">
    <string>Влево</string>
   </property>
  </action>
  <action name="rus_eng">
   <property name="text">
    <string>rus+eng   Ctrl+D</string>
   </property>
  </action>
  <action name="rus">
   <property name="text">
    <string>rus            Ctrl+R</string>
   </property>
  </action>
  <action name="eng">
   <property name="text">
    <string>eng           Ctrl+E</string>
   </property>
  </action>
  <action name="tr_eng">
   <property name="text">
    <string>на русский</string>
   </property>
  </action>
  <action name="tr_rus">
   <property name="text">
    <string>на английский</string>
   </property>
  </action>
  <action name="act_resize">
   <property name="text">
    <string>Обрезать</string>
   </property>
  </action>
  <action name="right_90">
   <property name="text">
    <string>Повернуть вправо</string>
   </property>
  </action>
  <action name="left_90">
   <property name="text">
    <string>Повернуть влево</string>
   </property>
  </action>
  <action name="paste">
   <property name="text">
    <string>Вставить             Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
